doc/user_guide/display_frontends.rst

```diff
@@ -135,10 +135,33 @@ Optionally, for offline rendering, you can use the mimetype renderer::
     # Optional in VS Code
     alt.renderers.enable('mimetype')
 
+.. _display_dashboards:
+
+Dashboards
+----------
+Altair is compatible with common Python dashboarding packages. Some of them even provide support for reading out :ref:`parameters <user-guide-interactions>` from the chart.
```

Comment:
I would say "Many" instead of "Some" after all the fantastic recent progress on this front via JupyterChart and dash_vega_components =)

Reply:
Fully agree! :)